packages/augur-ui/src/modules/events/actions/log-handlers.ts

```diff
@@ -405,6 +405,14 @@ export const handleBulkOrdersLog = (data: {
   }
 };
 
+export const handleDBUpdatedZeroXOrders = (log: StoredOrder) =>
```

Comment:
if UI is processing bulk orders this isn't needed.

Reply:
Agreed, there should be zero changes to which events are generated or listened to.

As an optimization we can allow each SDK consumer to subscribe to the relevant events in the future.

Reply:
Good catch. I missed this when I switched to bulk orders. Deleted.